hawkbit-rest/hawkbit-mgmt-resource/src/test/java/org/eclipse/hawkbit/mgmt/rest/resource/MgmtSoftwareModuleResourceTest.java

```diff
@@ -588,6 +591,10 @@ public void invalidRequestsOnArtifactResource() throws Exception {
 
         mvc.perform(delete("/rest/v1/softwaremodules/{smId}/artifacts", sm.getId())).andDo(MockMvcResultPrinter.print())
                 .andExpect(status().isMethodNotAllowed());
+
+        // SM soft deleted
+        mvc.perform(get("/rest/v1/softwaremodules/{smId}/artifacts/{artifactId}/download", smSoftDeleted.getId(),
+                artifactSoftDeleted.getId())).andDo(MockMvcResultPrinter.print()).andExpect(status().isGone());
```

Comment:
thanks for adding the test for GONE status! Could you please add another one (or extend the `getArtifact()` test) to verify no _download_ link is provided in case of soft deletion? Thanks!

Reply:
done